### PR TITLE
Update the egrep and service accounts.

### PR DESCRIPTION
**Pull request by incfly**
Changed the service account output and the egrep format (not sure why the old one does not work)

Test against latest build of 1.4 release, `istio-1.4-alpha.5385e895f4e045981a4369826389ea0fdbb2b9e6`.

### CODE REVIEW

**Comment by incfly** [comment]
This is after I tried to install with latest 1.4 daily build and istioctl manifest installation. Seems like the grep regex format needs to be updated and the service account we use for the sleep changed as well.

File: content/en/docs/tasks/security/authentication/auto-mtls/index.md [hunk 1]
```diff
@@ -126,13 +126,14 @@ You can verify setup by sending an HTTP request with `curl` from any `sleep` pod
 For example, here is a command to check `sleep.full` to `httpbin.full` reachability:
 
 {{< text bash >}}
-$ kubectl exec $(kubectl get pod -l app=sleep -n full -o jsonpath={.items..metadata.name}) -c sleep -n full -- curl http://httpbin.full:8000/headers  -s  -w "response %{http_code}\n" | egrep 'Client-Cert.*sa.default|response.*$' -o
-Client-Cert": "By=spiffe://cluster.local/ns/full/sa/default
+$ kubectl exec $(kubectl get pod -l app=sleep -n full -o jsonpath={.items..metadata.name}) -c sleep -n full -- curl http://httpbin.full:8000/headers  -s  -w "response %{http_code}\n" | egrep -o 'URI\=spiffe.*sa/[a-z]*|response.*$'
```
Comment (diemtvu) [inline content/en/docs/tasks/security/authentication/auto-mtls/index.md]: Or may be just grep "spiffe" to avoid other change in the future.

Comment (incfly) [reply content/en/docs/tasks/security/authentication/auto-mtls/index.md]: The actual header is 
```
    "X-Forwarded-Client-Cert": "By=spiffe://cluster.local/ns/full/sa/httpbin;Hash=22c9623e6aac2e31f8720678d0816c8f43be3534b3c0094a0f6c21ce3ec90020;Subject=\"\";URI=spiffe://cluster.local/ns/full/sa/sleep"
```

Both client and server certs, kinda of verbose and confusing.

Comment (diemtvu) [reply content/en/docs/tasks/security/authentication/auto-mtls/index.md]: Hmm, why it show the server SA. May be it's worth to check, outside the scope of this PR

Comment (incfly) [reply content/en/docs/tasks/security/authentication/auto-mtls/index.md]: The second part is the client SA.

**Pull request merged**